utils/build-presets.ini

```diff
@@ -1062,7 +1062,7 @@ test-installable-package
 reconfigure
 
 swift-install-components=compiler;clang-builtin-headers;stdlib;swift-syntax;sdk-overlay;license;sourcekit-xpc-service;swift-remote-mirror;swift-remote-mirror-headers
-llvm-install-components=llvm-cov;llvm-profdata
+llvm-install-components=llvm-cov;llvm-profdata;install-IndexStore
```

Comment:
Shouldn't that be `IndexStore` and not `install-IndexStore`?

Reply:
Good catch!

Reply:
Oooops, my bad.